app/src/components/Kontakt.vue

```diff
@@ -2,28 +2,28 @@
   <div id="kontakt" class="container-fluid col-sm-offset-4 col-sm-4">
     <div class="row">
       <div class="col-sm-12">
-        <h2>Kundensupport für Durchflussmessgeräte</h2>
+        <h2>{{ $t("Kontakt.Kundensupport_fuer_Durchflussmessgeraete") }}</h2>
```

Comment:
Punkt 1: Warum sollte der Übersetzungs-Key gleich dem deutschen Text sein?
Das macht in meinen Augen das Übersetzen schwieriger für nicht-deutschsprachige, zum Anderen sind die Keys extrem lang.

Wieso nicht einfach so etwas wie `$t("Kontakt.headline")` ?